src/commands/tileindex-validate/tileindex.validate.ts

```diff
@@ -353,32 +348,24 @@ export function validateTiffAlignment(tiff: TiffLocation, allowedError = 0.015):
 }
 
 export function getTileName(x: number, y: number, gridSize: GridSize): string {
-  if (!(SHEET_MIN_X <= x && x <= SHEET_MAX_X)) {
-    throw new Error(`x must be between ${SHEET_MIN_X} and ${SHEET_MAX_X}, was ${x}`);
-  }
-  if (!(SHEET_MIN_Y <= y && y <= SHEET_MAX_Y)) {
-    throw new Error(`y must be between ${SHEET_MIN_Y} and ${SHEET_MAX_Y}, was ${y}`);
-  }
-
   const offsetX = Math.round(Math.floor((x - MapSheet.origin.x) / MapSheet.width));
   const offsetY = Math.round(Math.floor((MapSheet.origin.y - y) / MapSheet.height));
 
   // Build name
   const letters = Object.keys(SheetRanges)[offsetY];
   const sheetCode = `${letters}${`${offsetX}`.padStart(2, '0')}`;
-  if (gridSize === mapSheetTileGridSize) {
-    // Shorter tile names for 1:50k
-    return sheetCode;
-  }
+  // TODO: re-enable this check when validation logic
+  // if (!MapSheet.isKnown(sheetCode)) throw new Error('Map sheet outside known range: ' + sheetCode);
```

Comment:
When this should be enabled?

Reply:
from the conversation yesterday its better to leave it disabled until we fix the validation logic.